src/components/Atom/Atom.types.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ArgumentArray } from '../../shared/utilities';
+
+type OcAtomBaseProps<T> = {
```

Comment:
Update this to use OcBaseProps - perhaps combining this component and those props.

Reply:
This one is slightly different from what is in the codebase, hence I created a new one. Currently, atom base prop accepts a generic and extends it as HTML attributes. Many components have blindly extended spans/divs. This needs to be furthered discussed and iterated.